app/src/main/java/com/example/habitsmasher/ui/profile/ProfileFragment.java

```diff
@@ -33,18 +39,22 @@ public View onCreateView(@NonNull LayoutInflater inflater,
         User user = new User("1", "TestUser", "123@gmail.com", "123");
 
         SharedPreferences sharedPref = getContext().getSharedPreferences(USER_DATA_PREFERENCES_TAG, Context.MODE_PRIVATE);
-        user.setUsername(sharedPref.getString("username", "user"));
+        user.setUsername(sharedPref.getString(USERNAME_SHARED_PREF_TAG, "user"));
+        String currentUserId = sharedPref.getString(USER_ID_SHARED_PREF_TAG, "id");
 
         // get the UI elements
         TextView usernameTextView = view.findViewById(R.id.username);
-        TextView followers = view.findViewById(R.id.number_followers);
-        TextView following = view.findViewById(R.id.number_following);
+        _numberOfFollowers = view.findViewById(R.id.number_followers);
+        _numberOfFollowing = view.findViewById(R.id.number_following);
         FloatingActionButton logoutButton = view.findViewById(R.id.logout_button);
 
         // set the UI elements
+        UserDatabaseHelper userDatabaseHelper = new UserDatabaseHelper(currentUserId,
+                                                                       _numberOfFollowers,
+                                                                       _numberOfFollowing);
         usernameTextView.setText("@" + user.getUsername());
-        followers.setText(String.valueOf(user.getFollowerCount()));
-        following.setText(String.valueOf(user.getFollowingCount()));
+        userDatabaseHelper.setFollowingCountOfUser();
+        userDatabaseHelper.setFollowerCountOfUser();
```

Comment:
These lines are responsible for setting the follower/following count on the profile page